extras/src/main/scala/com/avast/clients/rabbitmq/extras/testing/testing.scala

```diff
@@ -99,11 +82,11 @@ package object testing {
     }
   }
 
-  private def fkFToSyncIO[F[_]](implicit F: Effect[F]): FunctionK[F, SyncIO] = new FunctionK[F, SyncIO] {
+  private[rabbitmq] def fkFToSyncIO[F[_]](implicit F: Effect[F]): FunctionK[F, SyncIO] = new FunctionK[F, SyncIO] {
```

Comment:
Why are these things needed? If the client is really "finally tagless" than you can instantiate it to any effect data type you want so why would you do that and immediately convert it to some other data type?

Reply:
And I see it's supposed to be for testing but even for that I don't understand it.

Reply:
Yes. This is supposed to be suitable for testing. In past, it offered `RabbitMQConnection[Eval]`, but now I had to change it to `SyncIO`. The point was to introduce _blocking_ behavior (e.g. `producer.send` would not  have to be _run_ but would behave like a blocking call), unfortunately since migrating to `Resource` it's not possible (`Resource won't accept anything that's not `Bracket` etc.) So the current state is that you have `SyncIO` and you have to call `unsafeRunSync()` anyway :-/ so it maybe have lost it's sense. From my current POV, I have no problem deleting it, which you won't oppose I guess? :smiley: